pkg/systemd/reporting.jsx

```diff
@@ -295,7 +295,7 @@ class BusWorkflowRow extends React.Component {
                         /* GLib encodes errors for transport over the wire,
                          * but we don’t have a good way of decoding them without calling into GIO.
                          *
-                         * https://developer.gnome.org/gio/stable/gio-GDBusError.html#g-dbus-error-encode-gerror
+                         * https://developer-old.gnome.org/gio/stable/gio-GDBusError.html#g-dbus-error-encode-gerror
```

Comment:
Ugh, how poor -- the link is broken on https://www.gnome.org/technologies/ as well, I was hoping for some better replacement than this (which looks like it's going to be decomissioned soon)

Reply:
Right, I am also not convinced this is the best solution as it seems it is not going to stick around for too long. But that is where you get if you just navigate from https://developer.gnome.org